internal/executors/action/action.go

```diff
@@ -68,13 +68,9 @@ func (executor *Executor) Execute(meta execution.Metadata, metadata PlaybookStep
 				auth,
 				target,
 				metadata.Variables,
+				metadata.Step,
 				metadata.Agent)
 
-			if len(metadata.Step.OutArgs) > 0 {
```

Comment:
If we move the out_args in the CACAO v3 position (inside a command object) then I think we will need to rethink where these checks will be performed